remoto/log.py

```diff
@@ -12,7 +12,9 @@ def reporting(conn, result, timeout=None):
         try:
             received = result.receive(timeout)
             level_received, message = list(received.items())[0]
-            log_map[level_received](message.strip('\n'))
+            message = message.splitlines()
```

Comment:
I didn't think that the output could have ^M in the middle. The reason for why the `Reading database...` is still having `^M` in there is because it is not supposed to be a new line, but rather, a line that will get replaced as the percentage grows. This is a terminal emulator behavior to show progress.

I don't see it correct to split the lines to convert that into proper lines.

You pasted some log output, but I still don't see unit tests. Would you mind adding them to this pull request to demonstrate the behavior (and fix)?